### ISSUE TITLE
Weird indentation of lines in web mode.

### ISSUE THREAD

**Issue by norcalli** [syl20bnr/spacemacs#5646]
#### Description
After editing a line with `cc` and in a few other ways, like expanding a snippet (which has it's own separate problem with `$1` not being replaced occasionally), a line will indent beyond the fill column for no reason, but return once I put my cursor on it. The only way to fix it is by `k J <spc>jj` to join it with the line above and then split again.

#### Reproduction guide
- Start Emacs
- Open a javascript file, edit a line with `cc`, occasionally have it expand out strangely.

*Observed behaviour:*
![image](https://cloud.githubusercontent.com/assets/296363/14190735/236a72d2-f749-11e5-859d-be2b368c3f11.png)

*Expected behaviour:*
![image](https://cloud.githubusercontent.com/assets/296363/14190743/3041bb6e-f749-11e5-94e5-ec855c1f380d.png)

#### System Info
- OS: darwin
- Emacs: 24.5.1
- Spacemacs: 0.105.14
- Spacemacs branch: master (rev. a298a5a)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(emacs-lisp git lua scala sql osx markdown html colors syntax-checking version-control)
```

**Comment by TheBB** [comment]
This is likely the same problem as https://github.com/syl20bnr/spacemacs/issues/1950

The upstream bug report is here: https://github.com/alpaker/Fill-Column-Indicator/issues/46

FCI apparently has problems in certain modes, so your best bet is to disable it. The upstream bug has been open for ages so I don't see a fix in the near future.

In the meantime I'm closing this as a duplicate.

**Comment by norcalli** [comment]
Thank you! At least it's a solution.





On Thu, Mar 31, 2016 at 2:33 PM -0700, "Eivind Fonn" <notifications@github.com> wrote:












Closed #5646.
